### PR TITLE
Feature/154/new email confirm modal

### PR DESCRIPTION
**Pull request by greentoss**
crated modal, that pops up after email confirmation

### CODE REVIEW

File: src/components/email-confirm-modal/email-confirm-modal.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+const EmailConfirmModal = ( { confirmToken } ) => {
+  console.log( confirmToken , 'confirmToken inside EmailConfirmModal')
+  const [open, setOpen] = useState(true)
+  const [message, setMessage] = useState()
+  const [statusImage, setStatusImage] = useState(imgSuccess)
+  const [isLoading, setIsLoading] = useState(false)
+  const { setModal } = useContext(ModalContext)
+  const { t } = useTranslation()
+
+  //TODO: useAxios - u need to check what has come from confirmToken
```
Comment (greentoss) [inline src/components/email-confirm-modal/email-confirm-modal.js:25]: here i wait for backend part of email confirmation
where i will send query to backand and will get status: error or ok

File: src/components/email-confirm-modal/email-confirm-modal.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    return (
+      <Box sx={ style.box }>
+        <Box>
+          <Box
+            alt="email-reject-icon"
+            component="img"
+            src={ imgReject }
+          />
+          <Typography
+            component="h2" data-testid='reject-message' id="modal-modal-title"
```
Comment (Yevhendzyuba) [inline src/components/email-confirm-modal/email-confirm-modal.js:44]: For what you use id="modal-modal-title" ?

In line 73 you also have this id, but it never used in your code.

And i also think about naming

File: src/components/email-confirm-modal/email-confirm-modal.js [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    return (
+      <Box sx={ style.box }>
+        <Box>
+          <Box
+            alt="email-confirm-icon"
+            component="img"
+            src={ imgSuccess }
+          />
+          <Typography
+            component="h2" data-testid='confirm-message' id="modal-modal-title"
```
Comment (Yevhendzyuba) [inline src/components/email-confirm-modal/email-confirm-modal.js:73]: Here you use the same id

File: src/components/email-confirm-modal/email-confirm-modal.js (hunk 2 above)
Comment (greentoss) [reply src/components/email-confirm-modal/email-confirm-modal.js]: fixed

File: src/components/email-confirm-modal/email-confirm-modal.js (hunk 3 above)
Comment (greentoss) [reply src/components/email-confirm-modal/email-confirm-modal.js]: fixed

**Review by maarekua — APPROVED** [review]

**Pull request merged**